### DISCUSSION TITLE
Arabic letters don't show in epub file

### DISCUSSION THREAD

**Discussion by brandosz** [sumatrapdfreader/sumatrapdf#3383] [Q&A]
Hi all, I just downloaded an epub file which contains Arabic letters. However they don't show correctly, just the squares:
![image](https://user-images.githubusercontent.com/128929657/227735920-cb050865-fcd0-4ac6-8ade-abcbecf41d03.png)

Calibre can handle them well: 
![image](https://user-images.githubusercontent.com/128929657/227735984-284b02ad-e606-4ac7-8736-04e61ab76280.png)

Any idea on this? (Win 10, 19044.2486 Simplied Chinese; Sumatra V3.4.6 64-bit ) Many thanks in advance.

**Comment by kjk** [comment]
Most likely it's fixed in latest pre-release: https://www.sumatrapdfreader.org/prerelease There was an issue with missing arabic fonts which I fixed recently

**Reply by brandosz** [reply]
Hi, thanks for your prompt response. Just had a quick test of the pre-release, and the squares are still there. However, i noticed that some letters like "ḥ" "ḍ" now can display correctly compared to V3.4.6. 

![image](https://user-images.githubusercontent.com/128929657/227737199-e0a320a0-ef85-4d2e-9f2e-fb30efecfc9b.png)
